migration/registry/from-mi-1.0.0-to-later/README.md

```diff
@@ -0,0 +1,37 @@
+#Registry Migration Tool for WSO2 Micro Integrator
+
+This tool creates metadata files for registry resources. 
+Metadata files which are used to store media type of the resource were not available in the MI 1.0.0 release.
+The tool infers media types from the file extensions of the resource files.
+
+##How to run the tool
+
+1. Build RegistryMigrator using maven 
+
+`mvn clean package`
+
+2. Run the jar file providing the path to MI distribution 
+
+`java -jar target/RegistryMigrator-1.0.jar <Absolute_Path_To_MI_HOME>`
```

Comment:
Shall we point to the registry location instead of the product location?

This is due to the usage of shared file systems for the registry.